.github/workflows/jazzy-coverage-build.yml

```diff
@@ -0,0 +1,36 @@
+name: Coverage Build - Jazzy
+on:
+  workflow_dispatch:
+  # TODO(anyone) activate when branched for Jazzy
+  # push:
+  #   branches:
+  #     - master
+  #   paths:
+  #     - '**.hpp'
+  #     - '**.h'
+  #     - '**.cpp'
+  #     - '.github/workflows/jazzy-coverage-build.yml'
+  #     - '**.yaml'
+  #     - '**/package.xml'
+  #     - '**/CMakeLists.txt'
+  #     - 'codecov.yml'
+  # pull_request:
+  #   branches:
+  #     - master
+  #   paths:
+  #     - '**.hpp'
+  #     - '**.h'
+  #     - '**.cpp'
+  #     - '.github/workflows/jazzy-coverage-build.yml'
+  #     - '**.yaml'
+  #     - '**/package.xml'
+  #     - '**/CMakeLists.txt'
+  #     - 'codecov.yml'
+
+jobs:
+  coverage_jazzy:
+    uses: ros-controls/ros2_control_ci/.github/workflows/reusable-build-coverage.yml@master
+    secrets: inherit
+    with:
+      ros_distro: jazzy
+      container: ubuntu:24.04
```

Comment:
With https://github.com/ros-controls/ros2_control_ci/pull/72 that line should probably be removed.

(As should the other coverage build jobs)

Reply:
done